### ISSUE TITLE
Bluesky "get token" button returns user to login page

### ISSUE THREAD

**Issue by mathewi** [snarfed/bridgy#1864]
After logging in to Bluesky through Bridgy, I click the "get token" button and it returns me to the login page. After logging in again, I click the button again and the same behavior occurs.

**Comment by snarfed** [comment]
Hey Mathew! Hmm, sorry for the trouble! After you click the button and log in, it should show your token in a yellow banner at the top of the screen. Is that banner not appearing?

Here's an example:

<img width="1279" alt="Untitled" src="https://github.com/user-attachments/assets/be75e1e3-497d-43a6-9ca8-b6f946cf23d2" />

**Comment by mathewi** [comment]
No, the banner appeared with the token when I did the same for Mastodon,
but when I clicked the "get token" on the Bluesky page after logging in it
just took me back to login page. I tried several times and the same thing
happened.

**Comment by snarfed** [comment]
So odd! Looking at Bridgy's logs, it saw that you logged in successfully three times this morning - 07:29:13, 07:29:41, and 07:33:22 PT - and it redirected you back to your user page and showed the banner each time. Not sure what's going on otherwise, hrm.

Regardless, your Micropub token for Bluesky on Bridgy is just your app password, so feel free to try that!

**Comment by mathewi** [comment]
Great, thanks! I will try it again in a different browser, but failing that
I will just use the app password

**Comment by snarfed** [comment]
Bridgy's error messages depend on cookies, so one common cause of them not appearing is ad/cookie blocker extensions. That's my best guess as to what happened here. I don't have an obvious plan or strategy for getting around that, other than rewriting big chunks of Bridgy to work differently, so I'm going to tentatively close this. Feel free to reopen if you disagree!
